kDriveActionExtension/Info.plist

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>$(DEVELOPMENT_LANGUAGE)</string>
+	<key>CFBundleDisplayName</key>
+	<string>$(PRODUCT_NAME)</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>$(PRODUCT_NAME)</string>
+	<key>CFBundlePackageType</key>
+	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0</string>
+	<key>CFBundleVersion</key>
+	<string>1</string>
+	<key>NSExtension</key>
+	<dict>
+		<key>NSExtensionAttributes</key>
+		<dict>
+			<key>NSExtensionActivationRule</key>
+			<string>TRUEPREDICATE</string>
```

Comment:
<div class="js-discussion js-socket-channel ml-0 pl-0 ml-md-6 pl-md-3" data-channel="eyJjIjoibWFya2VkLWFzLXJlYWQ6NTg0MzA0NCIsInQiOjE2MjEyNzU5MTh9--ea6f2bdc2eb58ac88c3af7700800f4937fb91e14f38ec9896cdd0891468b24e9" data-channel-target="MDExOlB1bGxSZXF1ZXN0NjMxNDkzMTA1" style="box-sizing: border-box; margin-left: 40px !important; padding-left: 16px !important; color: rgb(36, 41, 46); font-family: -apple-system, system-ui, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="js-timeline-item js-timeline-progressive-focus-container" data-gid="MDE3OlB1bGxSZXF1ZXN0UmV2aWV3NjYxMjc1ODQ1" style="box-sizing: border-box;"><div class="
            is-rejected
            
            is-writer" style="box-sizing: border-box;"><div id="pullrequestreview-661275845" class="js-comment js-updatable-content js-socket-channel js-targetable-element" data-gid="MDE3OlB1bGxSZXF1ZXN0UmV2aWV3NjYxMjc1ODQ1" data-channel="eyJjIjoicHVsbF9yZXF1ZXN0X3Jldmlldzo2NjEyNzU4NDUiLCJ0IjoxNjIxMjc1OTE4fQ==--e602e1f1aa39cfdc1e1da8ec731d95818dcfb3009116b98f992ecdc7f738dfe7" data-url="/_render_node/MDE3OlB1bGxSZXF1ZXN0UmV2aWV3NjYxMjc1ODQ1/pull_request_reviews/body?variables%5BdeferredCommentActions%5D=false" style="box-sizing: border-box;"><div class="TimelineItem pt-0" style="box-sizing: border-box; padding: 16px 0px; position: relative; display: flex; margin-left: 16px;"><div class=" timeline-comment-group js-minimizable-comment-group js-targetable-element TimelineItem-body my-0 " id="pullrequestreview-661275845-body-html" style="box-sizing: border-box; margin-top: 4px; margin-bottom: 0px !important; min-width: 0px; max-width: 100%; color: var(--color-timeline-text); flex: 1 1 auto;"><div class="ml-n3 timeline-comment unminimized-comment comment previewable-edit js-task-list-container editable-comment js-comment timeline-comment--caret timeline-comment--caret-nw reorderable-task-lists current-user" data-body-version="995af9b49c8fc5c7ea604e9580445f0731247fba02b7f49f6fe5dc77bc55e2cc" data-unfurl-hide-url="/content_reference_attachments/hide" style="box-sizing: border-box; margin-left: -16px !important; position: relative; color: var(--color-text-primary); background-color: var(--color-bg-primary); border-color: var(--color-box-border-info); border-top-style: ; border-top-width: ; border-right-style: ; border-right-width: ; border-bottom-style: ; border-bottom-width: ; border-left-style: ; border-left-width: ; border-image-source: ; border-image-slice: ; border-image-width: ; border-image-outset: ; border-image-repeat: ; border-radius: 6px;"><div class="edit-comment-hide" style="box-sizing: border-box;"><task-lists sortable="" style="box-sizing: border-box;">We can't use TRUEPREDICATE in production</task-lists></div><form class="js-comment-update" id="pullrequestreview-661275845-body-html-edit-form" action="https://github.com/Infomaniak/ios-kDrive/pull/34/reviews/661275845/update" accept-charset="UTF-8" method="post" style="box-sizing: border-box;"></form></div></div></div></div></div></div><div id="partial-timeline" class="js-timeline-marker js-socket-channel js-updatable-content" data-channel="eyJjIjoicHVsbF9yZXF1ZXN0OjYzMTQ5MzEwNSIsInQiOjE2MjEyNzU5MTh9--e8eef6f2bc374dae6deeb633187cd9a4ab2165aa4fc0c8939115d97af048f045" data-url="/_render_node/MDExOlB1bGxSZXF1ZXN0NjMxNDkzMTA1/pull_requests/unread_timeline?variables%5BdeferCommitBadges%5D=false&amp;variables%5BdeferStatusCheckRollups%5D=false&amp;variables%5BdeferredCommentActions%5D=true&amp;variables%5BhasFocusedReviewComment%5D=false&amp;variables%5BhasFocusedReviewThread%5D=false&amp;variables%5BtimelinePageSize%5D=30&amp;variables%5BtimelineSince%5D=2021-05-17T18%3A24%3A30Z" data-last-modified="Mon, 17 May 2021 18:24:30 GMT" data-gid="MDExOlB1bGxSZXF1ZXN0NjMxNDkzMTA1" style="box-sizing: border-box;"></div></div><div class="discussion-timeline-actions" style="box-sizing: border-box; background-color: var(--color-bg-canvas); border-top: 2px solid var(--color-border-primary); color: rgb(36, 41, 46); font-family: -apple-system, system-ui, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div id="partial-pull-merging" class="pull-merging js-pull-merging js-socket-channel js-updatable-content js-pull-refresh-on-pjax " aria-live="polite" data-channel="eyJjIjoicmVwbzozNTk0MTU2MTM6YnJhbmNoOmFjdGlvbi1leHRlbnNpb24iLCJ0IjoxNjIxMjc1OTE4fQ==--60f7c5af42aa6fd14eee20e229b5d192073c2000bd8d86ce5d0ac624fe3b3e67 eyJjIjoicmVwbzozNTk0MTU2MTM6YnJhbmNoOm1hc3RlciIsInQiOjE2MjEyNzU5MTh9--0fe24475287fdee829dce5892e5934896ca23531b80fe68e4531807cfdd0de5a eyJjIjoicmVwbzozNTk0MTU2MTM6Y29tbWl0OjBjZWNhNTExOThjYjI5YzAyNWUzZGZhZWUyOTY1ODZlY2RjYjgwYTYiLCJ0IjoxNjIxMjc1OTE4fQ==--986d0dfbe7f6dcaa522df2360f6197736daa5b5b2668383b214e838fea1776ff eyJjIjoiaXNzdWU6ODc3NTYyMzU4OnN0YXRlIiwidCI6MTYyMTI3NTkxOH0=--ab2dcf033af1f2ea0647c4ca896a93219a8e7a32b42c686be0ec356bf0a3814a eyJjIjoicHVsbF9yZXF1ZXN0OjYzMTQ5MzEwNTpyZXZpZXdfc3RhdGUiLCJ0IjoxNjIxMjc1OTE4fQ==--2b90f1c499ccabad7e722d23169effa140e68f817d524d3796c1ad4c4cf59146 eyJjIjoicHVsbF9yZXF1ZXN0OjYzMTQ5MzEwNTp3b3JrZmxvd19ydW4iLCJ0IjoxNjIxMjc1OTE4fQ==--adabca508ac4daa95bb873cb92f1618642daaa0d7f2a522796e3b2d8507f7a26" data-url="/Infomaniak/ios-kDrive/pull/34/show_partial?merge_type=merge&amp;partial=pull_requests%2Fmerging" style="box-sizing: border-box;"><br class="Apple-interchange-newline"></div></div>